modules/raft/src/main/java/org/apache/ignite/internal/raft/server/RaftServer.java

```diff
@@ -60,4 +61,11 @@
      * @return Local peer or null if the group is not started.
      */
     @Nullable Peer localPeer(String groupId);
+
+    /**
+     * Returns a set of started partition groups.
```

Comment:
This raft groups doesn't necessarily partition's group, it might be meta storage, I would remove "partition"